File: projekatWeb2/src/app/incidenti/incident.ts
export class Incident {
  id: number;
  type: string;
  priority:string;
  confirmed:boolean;
  status:string;
  eta:Date;
  description:string;
  ata:Date;
  outageTime:Date;
  etr:Date;
  affectedCustommers:number;
  calls:number;
  voltage:string;
  scheduledTime:Date;
  userCreated: string;



  constructor() {
    this.id = 0;
    this.type = '';
    this.priority='';
    this.confirmed=false;
    this.status='Draft';
    this.eta=new Date();
    this.description='';
    this.ata=new Date();
    this.outageTime=new Date();
    this.etr=new Date();
    this.affectedCustommers=0;
    this.calls=0;
    this.voltage='';
    this.scheduledTime=new Date();
    this.userCreated = '';

  
  }
}

File: Backend/Backend/Models/Incident.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Backend.Models
{
    public class Incident
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int id { get; set; }
        public string Type { get; set; }
        public string Priority { get; set; }
        public string Confirmed { get; set; }
        public string Status { get; set; }
        public DateTime ETA { get; set; }
        public string Description { get; set; }
        public DateTime ATA { get; set; }
        public DateTime OutageTime { get; set; }
        public DateTime ETR { get; set; }
        public int AffectedCustommers { get; set; }
        public int Calls { get; set; }
        public string Voltage { get; set; }
        public DateTime ScheduledTime { get; set; }
    }
}
